.circleci/config.yml

```diff
@@ -312,6 +313,75 @@ jobs:
     <<: *test-defaults
     environment:
       - TEST_TARGET=sanity
+  build-waterfall:
```

Comment:
Bikeshedding what to use instead of "waterfall" here, I'd say either "backend", "wasm_backend", or "upstream"

Reply:
Sounds good, emsdk already has it that way anyhow, updating.